pa11y/README.md

```diff
@@ -0,0 +1 @@
+# pa11y testing for JupyterLab and friends
```

Comment:
Is "friends" the same as other front-ends?

Reply:
At this point the only other frontend component under test is `lumino`. Very likely, we'll see PRs through `ipython`, `xeus-python`, `jupyter_server`, `jupyterhub`, `nbgrader`, `ipywidgets`, and `jupyterlab_server` evaluated here as well, as they each create at least _some_ DOM.

As to other frontend: if the other front-ends have maintainers that wish to PR, hooray! But these things are out of _my_ personal, free time scope to write and/or support. Some challenges I see to supporting:

- `notebook`: none, really, other than developer time
- `spyder`: I don't know the qt accessibility testing tools
- `nteract`: i'd _really_ like to keep a full Electron build out of this
- `nteract-on-jupyter`: recent nteracts seem to require per-release (PR?) knowledge like `Click on element .nsojdfbijabsjif-1231`
  - i had to drop recent nteracts-on-jupyter from `robotframework-jupyterlibrary` for this reason
  - pa11y doesn't (to my knowledge) support XPath queries, which could save _some_ of it with extreme jumping jacks
  - if it _was_ fully marked up with `role`s and `aria-labels`, this would be a lot easier

Reply:
Anyhow, i took out references to any specific project, and actually put in some of the content i had kicking around in my mind